packages/core/src/browser/style/quick-title-bar.css

```diff
@@ -0,0 +1,53 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.theia-quick-title-container {
+    display: flex;
+    flex-direction: row;
+    font-size: 11px;
+    padding: 3px 3px;
```

Comment:
please use defined variables for padding that everyhing sclaes nicely when someone customize, i.e. `padding: calc(var(--theia-ui-padding)/2))`

Reply:
done